mitodl/mit-open
# Prolearn ETL

# What are the relevant tickets?
Closes #151 

# Description (What does it do?)
- Migrates and refactors the Prolearn ETL pipeline from `course_catalog` to `learning_resources`, and adds new fields to the `LearningResourceOfferor` model (somewhat relevant to https://github.com/mitodl/mit-open/issues/158). Changes in 1st commit)
- Since this was the last ETL pipeline that needed to be migrated (not doing videos), deletes everything from `course_catalog` except the bare minimum needed to keep the `search` app functioning & tests passing. Changes in 2nd commi.
- Adds the prolearn ETL pipeline to the list of scheduled celery tasks (3rd commit)


# How can this be tested?
- Run `docker compose up`
- Run `./manage.py update_offered_by`
- Run `./manage.py backpopulate_prolearn_data`
- You should have a bunch of courses created afterward with `etl_source=prolearn` and various offered_by/platform values (Sloan Executive Education, Schwartzman Computing College, CSAIL, Professional Education, Center for Transportation & Logistics)
